Example/bottomSheet/index.js

```diff
@@ -99,34 +97,43 @@ export default class BottomSheet extends Component {
   };
   render() {
     return (
-      <View style={styles.container}>
-        <TapGestureHandler
-          maxDurationMs={100000}
-          id="masterdrawer"
-          maxDeltaY={this.state.lastSnap - SNAP_POINTS_FROM_TOP[0]}>
-          <View style={StyleSheet.absoluteFillObject}>
+      <TapGestureHandler
```

Comment:
I'm curious what the point of this TapGestureHandler is. It would seem to be related to the scrollview interaction, but I can't quite figure out the logic. Any tips?